mypy/checkexpr.py

```diff
@@ -1439,6 +1439,23 @@ def check_for_extra_actual_arguments(self,
                                                                             context)
                                 is_unexpected_arg_error = True
                         ok = False
+                elif isinstance(actual_type, Instance) and \
```

Comment:
Style nit: We prefer extra parentheses instead of `\` to deal with expressions split into multiple lines.